RELEASE_PROCEDURE.md

```diff
@@ -88,6 +66,42 @@
      anaconda upload -u stellargraph /path/to/conda-package.tar.bz2
      ```
 
+5. **Make release on GitHub**
+
+    After successfully publishing to PyPi and Conda, we now want to make the release on GitHub.
+
+   - Push `master` branch
+    ```shell
+    git push --follow-tags origin master
+    ```
+   - Go to the tags on the GitHub stellargraph homepage: https://github.com/stellargraph/stellargraph/tags
+   - Next to the release tag, click the “...” button and select “create release”
+   - Add the title and text of the metadata: a title “Release X.X.X” and text copied from the changelog is good practice
+   - Click “Publish release”
+
+6. **Get `develop` into correct state for next development version**
+
+    We want the merge any of the changes made during the release back into `develop`, and make sure the new version in `develop` is correct.
+
+   - Switch to `develop` branch:
+     ```shell
+     git checkout develop
+     ```
+   - Increase the version: in `stellargraph/version.py`, change version from `X.X.X` to `X.X+1.Xb`. E.g. `__version__ = "0.2.0"` to `__version__ = "0.3.0b"`. (To stay consistent we use `b` to indicate “beta”, but python will accept any string after the number. In semantic versioning: first number for major release, second number for minor release, third number for hotfixes.)
```

Comment:
```suggestion
   - Increase the version: in `stellargraph/version.py` and `meta.yaml`, change version from `X.X.X` to `X.X+1.Xb`. E.g. `__version__ = "0.2.0"` to `__version__ = "0.3.0b"`. (To stay consistent we use `b` to indicate “beta”, but python will accept any string after the number. In semantic versioning: first number for major release, second number for minor release, third number for hotfixes.)
```

I guess this will also need to change to `git add stellargraph/version.py meta.yaml` in the next line too.

Reply:
Do we need to update this in develop? It only gets used during a release specifically for publishing to conda cloud, i.e. `meta.yaml` with a `x.x.xb` version will never be used, until it gets updated to `x.x.x` for the next release again.

Reply:
I thought it would be good to in case someone wants to install the Conda package from GitHub... but it turns out that's not how Conda works (e.g. https://stackoverflow.com/a/19071214/1256624 and the other answers here), and one generally just does a pip install inside the Conda environment. As such, yeah, I agree with you. 👍